python/tests/backends/test_oqc.py

```diff
@@ -0,0 +1,145 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+import os
+import sys
+import time
+
+import pytest
+from multiprocessing import Process
+
+import cudaq
+from cudaq import spin
+
+try:
+    from utils.mock_qpu.oqc import startServer
+except:
+    print("Mock qpu not available, skipping OQC tests.")
+    pytest.skip("Mock qpu not available.", allow_module_level=True)
+
+# Define the port for the mock server
+port = 62444
+
+
+def assert_close(got) -> bool:
+    return got < -1.1 and got > -2.2
+
+
+@pytest.fixture(scope="session", autouse=True)
+def startUpMockServer():
+
+    credsName = '{}/FakeConfig.config'.format(os.environ["HOME"])
+    f = open(credsName, 'w')
+    f.write('email: {}\npassword: {}\ntime: 0'.format("email@email.com", "password"))
+    f.close()
+
+    # Set the targeted QPU
+    cudaq.set_target('oqc', url=f'http://localhost:{port}', credentials=credsName)
```

Comment:
This suggested change should fix the Python mock server test failures in the CI. It looks like my prior updates to authorization for the real servers broke these tests.
```suggestion
    # Set the targeted QPU
    cudaq.set_target('oqc',
                     url=f'http://localhost:{port}',
                     email="email@mail.com",
                     password="password")
```